task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -394,8 +394,44 @@ spec:
 
         if [ -n "${TARGET_STAGE}" ]; then
           BUILDAH_ARGS+=("--target=${TARGET_STAGE}")
+
```

Comment:
I assume this is a start and prepareation fo contextual SBOMs. I am not really sure if adding more bash code for simple logic like this is way to go forward. With this code it can't be tested properly and we can't verify if the logic is correct. How about wrapping up the logic into some Python scirpts that handles the overall contextual SBOM logic.

Reply:
I will reimplement this. Closing this PR.